v3/mothwing/mothwing.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "MOTHWING",
+  "vendorId": "0x4D77",
+  "productId": "0x0001",
+  "matrix": {"rows": 8, "cols": 6},
+  "customKeycodes": [
+    {
+      "name": "Zenkaku/Hankaku",
+      "title": "Toggle Zenkaku Hankaku",
+      "shortName": "ZenHan"
+    }
+  ],
```

Comment:
I dont' see a custom keycode in your QMK firmware

Reply:
oh thanks